### ISSUE TITLE
Scaffold-DbContext Computed Column

### ISSUE THREAD

**Issue by pablotdv** [aspnet/EntityFramework#7918]
When I run the "Scaffold-DbContext" command on an existing database. I'm having problems with the calculated columns.

The script of my Table

```
CREATE TABLE [dbo].[Caixas](
	[CaixaId] [uniqueidentifier] NOT NULL,
	[Percentual] [decimal](18, 2) NOT NULL,
	[Quantidade] [decimal](18, 2) NOT NULL,
	[Valor] [decimal](18, 2) NOT NULL,
	[ValorDesconto]  AS (([Valor]*[Quantidade])/([Percentual]/(100))),
 CONSTRAINT [PK_Caixas] PRIMARY KEY CLUSTERED 
(
	[CaixaId] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
) ON [PRIMARY]

```

After running the command 

`Scaffold-DbContext "Server=.\SQLExpress;Database=TestComputedColumn;Trusted_Connection=True;" Microsoft.EntityFrameworkCore.SqlServer -OutputDir Models2`

My OnModelCreating was as follows:
```
protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.Entity<Caixas>(entity =>
            {
                entity.HasKey(e => e.CaixaId)
                    .HasName("PK_Caixas");

                entity.Property(e => e.CaixaId).ValueGeneratedNever();

                entity.Property(e => e.Percentual).HasColumnType("decimal");

                entity.Property(e => e.Quantidade).HasColumnType("decimal");

                entity.Property(e => e.Valor).HasColumnType("decimal");

                entity.Property(e => e.ValorDesconto)
                    .HasColumnType("decimal")
                    .HasComputedColumnSql("[Valor]*[Quantidade])/([Percentual]/(100)")
                    .ValueGeneratedOnAddOrUpdate();
            });
        }
```
Note the attribute HasComputedColumnSql, where I have some missing parentheses.

The correct is  HasComputedColumnSql("**(**[Valor]*[Quantidade])/([Percentual]/(100)")

### Further technical details
EF Core version: netcoreapp1.1
Database Provider: (e.g. Microsoft.EntityFrameworkCore.SqlServer)
Operating system: 
IDE: (e.g. Visual Studio 2017)

**Comment by ajcvickers** [comment]
@lajones We may have some special handling for this in EF6; you might want to look at it for reference.

**Comment by pablotdv** [comment]
So I would have some tutorial explaining how I could do to debug the EF project code.

**Comment by ajcvickers** [comment]
@pablotdv I'm not sure exactly what you are asking, but if you mean building and working with the EF code yourself, then there some instructions here: https://github.com/aspnet/EntityFramework/wiki/Getting-and-Building-the-Code

**Comment by lajones** [comment]
Note: we decided not to trim. In fact we are now not interpreting the value we get from the metadata at all - we just put it into the `HasComputedColumnSql()` method (this also applies to the `HasDefaultValueSql()` method) exactly as it comes from the DB.

**Comment by lajones** [comment]
Fixed by PR #8217.
